tom_nonsidereal_airmass/templatetags/nonsidereal_airmass_extras.py

```diff
@@ -22,7 +31,8 @@ def nonsidereal_target_plan(context):
         plan_form = NonsiderealTargetVisibilityForm({
             'start_time': request.GET.get('start_time'),
             'end_time': request.GET.get('end_time'),
-            'airmass': request.GET.get('airmass')
+            'airmass': request.GET.get('airmass'),
+            'target': context['object'] if 'object' in context else context['target']
```

Comment:
Oh, one other thing--`NonsiderealTargetVisibilityForm` doesn't have a `target` field, so this will either break or do nothing. Is there a need for it in the form?